arkane/encorr/ae.py

```diff
@@ -134,6 +144,49 @@ def execute(self, output_file: str = None):
                 logging.warning('Could not write atom energies to database. Captured error:')
                 logging.warning(str(e))
 
+    def save_correlation_mat(self, path: str, labels: List[str] = None):
+        """
+        Save a visual representation of the parameter correlation matrix.
+
+        Args:
+            path: Path to save figure to.
+            labels: Parameter labels.
+        """
+        try:
+            import matplotlib.pyplot as plt
+        except ImportError:
+            return
```

Comment:
please add `logging.error('cannot import matplotlib, original error'+str(e)`

Reply:
And the same in BACs

Reply:
Added!